### ISSUE TITLE
Header missing in list-components

### ISSUE THREAD

**Issue by imicer** [IBM/cloud-pak-deployer#769]
**Describe the bug**
Following the steps to install CP4D5.0.0 on OCP 4.12 with the command:
https://ibm.github.io/cloud-pak-deployer/10-use-deployer/3-run/existing-openshift/

`./cp-deploy.sh env apply --accept-all-licenses -v`

It showed the following error:
```
TASK [cp4d-variables : set_fact] ***********************************************
Thursday 29 August 2024  13:41:26 +0000 (0:00:00.232)       0:01:58.022 *******
ok: [localhost]

TASK [cp4d-variables : Fail if there were undefined cartridges found] **********
Thursday 29 August 2024  13:41:26 +0000 (0:00:00.030)       0:01:58.053 *******
skipping: [localhost]

TASK [cp4d-variables : Add versions details from olm-utils] ********************
Thursday 29 August 2024  13:41:26 +0000 (0:00:00.025)       0:01:58.078 *******
fatal: [localhost]: FAILED! =>
  msg: |-
    The task includes an option with an undefined variable. The error was: {{ _list_components_json.list
      | selectattr('Component_name','equalto',item['olm_utils_name'])
      | list
      | first
      | default([])
    }}: 'dict object' has no attribute 'Component_name'

    The error appears to be in '/cloud-pak-deployer/automation-roles/50-install-cloud-pak/cp4d/cp4d-variables/tasks/main.yml': line 64, column 3, but may
    be elsewhere in the file depending on the exact syntax problem.

    The offending line appears to be:


    - name: Add versions details from olm-utils
      ^ here

PLAY RECAP *********************************************************************
localhost                  : ok=345  changed=34   unreachable=0    failed=1    skipped=217  rescued=0    ignored=0
```

**Expected behavior**
I can install CP4D 4.8.x successfully some days ago. Now I can't install CP4D5.0 with the same steps.

Could you please help take a look into the case?

**Comment by imicer** [comment]
@fketelaars Could you please help check it? Thanks.

**Comment by fketelaars** [comment]
@imicer I think this is a defect in th `list-components` command of the `olm-utils-v3` image. To avoid deployer crashing when the header is missing, I have adapted the code to supplement the header.

**Comment by fketelaars** [comment]
Fixed in the `v3.0.1` release.

**Comment by imicer** [comment]
@fketelaars Thanks for your fix. Now it can work. Thanks.
